connectors/source.py

```diff
@@ -272,7 +277,15 @@ def get_source_klass(fqn):
     return getattr(module, klass_name)
 
 
-def get_data_sources(config):
+def get_source_klasses(config):
     """Returns an iterator of all registered sources."""
     for name, fqn in config["sources"].items():
         yield get_source_klass(fqn)
+
+
+def get_source_klass_dict(config):
```

Comment:
makes me wonder if we don't want to add a `name` property to the source klass...

Reply:
We should. we actually have `name` property in Ruby.

Currently the `list` command uses `__doc__` of each source klass:

https://github.com/elastic/connectors-python/blob/c1baa3a0bdd74a173bedb545dad01df276e3faef/connectors/cli.py#L135


I can make the change in a separate PR.

But even we have `name` and `service_type` in each source klass, we still need a dict to quickly loop up the source klass

Reply:
PR is here: https://github.com/elastic/connectors-python/pull/413